tests/vxlan-non-zero-reserved-fields/test.yaml

```diff
@@ -0,0 +1,17 @@
+args:
+  - --set decoder.vxlan.enabled=true
+
+checks:
+  - filter:
+      count: 1
+      match:
+        event_type: flow
+        src_port: 51252
+        dest_port: 80
+        flow.pkts_toserver: 5
+        flow.pkts_toclient: 5
+  - filter:
+      count: 1
+      match:
+        event_type: flow
+        dest_port: 4789
```

Comment:
Since this change is to ensure that the received packets are properly parsed by Suricata, I think we could have a check for the stats here that show that:
```yaml
- filter:
      count: 1
      match:
        event_type: stats
        stats.decoder.vxlan: 10
```
This will require a minimum suricata.yaml